### DISCUSSION TITLE
First availability dates of GTFS-RT feeds?

### DISCUSSION THREAD

**Discussion by writezhe** [transitland/transitland#413] [Q&A] [answered]
Hi there, 

Is it possible to get (either through the API or through the Transit Land organization), a way to identify the date when different transit agencies first made GTFS-RT (realtime) feeds available?

I'm a hobbyist/academic working on a project where knowing when their URL or transit feed was first available and usable by the public would be quite helpful.

There is an archive of the static GTFS feeds (which is very helpful!), but I'm not sure if there's a similar info for realtime. Archive info here:
https://www.transit.land/news/2022/06/16/feed-version-archive-download

Thanks!
PS the availability date for the GBFS (realtime shared bike mobility feed) would also be helpful too

**Comment by drewda** [comment, marked as answer]
Regarding timing of adoption of static GTFS by agencies, see this paper which made use of Transitland data: https://findingspress.org/article/57722-predictors-of-early-adoption-of-the-general-transit-feed-specification

Regarding timing of adoption of GTFS Realtime, Transitland has less relevant information to offer. It is possible to use GitHub to explore the history of Transitland's directory of feeds: https://github.com/transitland/transitland-atlas By analyzing the git repository, it could be possible to determine the timing of when Transitland added listing for a given RT feed.

For GBFS, Transitland Atlas draws from a different git repository. The history of this git repository could be analyzed in a similar manner. The relevant file to analyze is called `systems.csv`: https://github.com/MobilityData/gbfs/commits/master/systems.csv

Good luck with your project.
